src/main/java/io/turntabl/employementprofilingsystem/Controllers/EmployeeController.java

```diff
@@ -200,14 +228,16 @@
         request.put("employee_address",editEmployee.getEmployee_address());
         request.put("employee_dev_level",editEmployee.getEmployee_dev_level());
         request.put("employee_status",editEmployee.getEmployee_status());
-
+        rootSpan.setTag("Employee to be edited data", request.toString());
         try{
             List<String> requiredParams = Arrays.asList(
                     "employee_id"
             );
             Map<String, Object> valid = parsor.validate_params(request,requiredParams);
+            rootSpan.setTag("employee_param_validation_status", valid.toString());
             if (valid.get("code").equals("00")){
                 Map<String, Object> updated_params = this.check_updated_params(editEmployee);
+                rootSpan.setTag("updated_employee_params_check", updated_params.toString());
```

Comment:
ditto

Reply:
so if traces support key/value pairs, shouldn't you just add the contents of the map to the trace? Can you do that?